packages/react-components/react-table/stories/Table/SingleSelect.stories.tsx

```diff
@@ -143,7 +143,7 @@ export const SingleSelect = () => {
     <Table aria-label="Table with single selection">
       <TableHeader>
         <TableRow>
-          <TableSelectionCell type="radio" hidden />
+          <TableSelectionCell type="radio" invisible hidden />
```

Comment:
```suggestion
          <TableSelectionCell type="radio" invisible />
```

oops forgot to remove hidden